src/pykeen/evaluation/evaluator.py

```diff
@@ -75,8 +75,20 @@ def __init__(
         batch_size: Optional[int] = None,
         slice_size: Optional[int] = None,
         automatic_memory_optimization: bool = True,
+        additional_filter_triples: Optional[MappedTriples] = None,
     ):
+        """Initialize the evaluator.
+
+        :param filtered: Should filtered evaluation be performed?
+        :param requires_positive_mask: Does the evaluator need access to the masks?
+        :param batch_size: >0. Evaluation batch size.
+        :param slice_size: >0. The divisor for the scoring function when using slicing
+        :param automatic_memory_optimization: Whether to automatically optimize the sub-batch size during
+            batch size during evaluation with regards to the hardware at hand.
+        :param additional_filter_triples: Additional true triples to filter out during filtered evaluation.
```

Comment:
This doesn't really give enough insight into how this should be used. Maybe a code example in the docstring under the parameters makes more sense (similar to the one I added in the Filtering tutorial page)

Reply:
Or include the one where you roll your own pipeline here